content/docs/authkem-abridged/index.md

```diff
@@ -94,6 +94,14 @@ This draft, to reach its full potential, will eventually require quantum-safe ce
 The transition to post-quantum cryptography will present a massive challenge to the PKI infrastructure and certificate issuers. We think it's not wise to wait for them, however. We especially think that this proposal could be a wake-up call that we might need other algorithms than just signature schemes in the PKI ecosystem.
 
 
+### How does this proposal relate to KEMTLS?
+
+[KEMTLS][KEMTLS] is an academic proposal to do a post-quantum TLS-like protocol without handshake signatures.
+In the academic paper, both key exchange and the authentication bits are considered.
+In the AuthKEM draft, we have isolated the authentication-via-KEM bits from the KEMTLS paper.
+We do not adress the ephemeral key exchange part of TLS, as we believe that, as a less controversial and more urgent subject, would be better discussed separately.
```

Comment:
I'd omit the language about ephemeral key exchange being "less controversial", and just focus on the fact that post quantum ephemeral key exchange can and should be treated independently of the authentication mechanism.